### PR TITLE
Add Admin UI for repository aliases

### PR DESCRIPTION
**Pull request by prazuma**
#104
- add repo_alias

### CODE REVIEW

**Comment by gimite** [comment]
Thanks for the pull request!

The title of the pull request should be more descriptive. It should be something like "Add Admin UI for repository aliases" instead of "Mikahashimoto add admin ui". Can you change the title?

File: app/resources/admin.html.template [hunk 1]
```diff
@@ -804,6 +804,27 @@
     </div>
   </fieldset>
 
+  <fieldset>
+    <legend>REPO</legend>
```
Comment (gimite) [inline app/resources/admin.html.template]: Write "Repository aliases" here instead.

File: app/resources/admin.html.template [hunk 2]
```diff
@@ -804,6 +804,27 @@
...
+    <div class="config">
+      <label for="repo_aliases">
+        REPO alias mapping.<br>
```
Comment (gimite) [inline app/resources/admin.html.template]: Use word "repository" instead of "REPO" here and below.

File: app/resources/admin.html.template [hunk 3]
```diff
@@ -804,6 +804,27 @@
...
+        e.g., If it is
+        <code>{"jp": "japan", "ht": "haiti"}</code>, REPO
+        alias to jp are interpreted as requests for "japan"
```
Comment (gimite) [inline app/resources/admin.html.template]: Can you clarify this explanation? For example,
e.g., If it is <code>{"jp": "japan", "ht": "haiti"}</code>, you can use http://google.org/personfinder/jp as an alias of http://google.org/personfinder/japan and the same for "ht". This is especially useful combined with the short URL e.g., http://g.co/pf/jp

File: app/resources/admin.html.template [hunk 4]
```diff
@@ -804,6 +804,27 @@
...
+        repository, and messages to ht are interpreted as requests
+        to "haiti" repository.<br>
+        JSON dictionary where the key is an international phone number and the
```
Comment (gimite) [inline app/resources/admin.html.template]: Update this line too. the key is not an international phone number here.

**Comment by prazuma** [comment]
applied comment

**Comment by prazuma** [comment]
apply comment

**Comment by prazuma** [comment]
apply Repository->repository

File: app/resources/admin.html.template [hunk 5]
```diff
@@ -804,6 +804,38 @@
...
+      <label for="repo_aliases">
+        Repository alias mapping.<br>
+        e.g., If it is
+        <code>{"jp": "japan", "ht": "haiti"}</code>, 
+	you can use 
+	<a href="http://google.org/personfinder/jp">
+	  http://google.org/personfinder/jp
+	</a>
+	 as an alias of
+	<a href="http://google.org/personfinder/jp">
```
Comment (gimite) [inline app/resources/admin.html.template]: Sorry one more minor fix. jp -> japan here.

**Comment by prazuma** [comment]
fix jp->japan

**Comment by gimite** [comment]
LGTM. Merged.

**Pull request merged**